pkgs/tools/security/enumerepo/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "enumerepo";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "trickest";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-PWWx6b+fttxKxMtuHAYPTeEsta0E6+IQ1DSKO6c7Jdc=";
+  };
+
+  vendorHash = "sha256-Dt3QS1Rm/20Yitgg4zbBcWQXV8mTlpNbzc/k4DaTuQc=";
+
```

Comment:
```suggestion

  ldflags = [
    "-s"
    "-w"
 ];
 
```

Reply:
This should reduces the closure size by removing debugging symbols, in the future this will be the default for `buildGoModule`